webpack.config.js

```diff
@@ -0,0 +1,37 @@
+//@ts-check
+"use strict";
+
+const path = require("path");
+
+/**@type {import('webpack').Configuration}*/
+const config = {
+	target: "node", // vscode extensions run in a Node.js-context 📖 -> https://webpack.js.org/configuration/node/
```

Comment:
Do we really need all these comments? They look redundant.

Reply:
Hahaha... yeah... I also just realize it... :smile: 

WIll remove that...